open-api/rest-catalog-open-api.yaml

```diff
@@ -1070,6 +1127,35 @@ paths:
       summary: List all view identifiers underneath a given namespace
       description: Return all view identifiers under this namespace
       operationId: listViews
+      parameters:
+        - name: pageToken
+          in: query
+          description:
+            Allows clients to make use of pagination by signaling to the service that they would
+            prefer requests to be paginated to a “reasonable” number of results.
+            
+            New Clients always start the request by sending a required empty “pageToken” e.g. GET /tables?continuationToken=””
+            Servers supporting pagination would recognize pagination is requested due to the presence of the pageToken
+            and honor the contracts specified above by returning a non-empty continuation token
+            if there are more results available, or an empty token or missing field indicating no more results.
+            Servers that are non-pagination aware will ignore the token and return all results as they previously did.
+            
+            Old clients should not be specifying the new parameters
+            For servers that support pagination, they would notice the lack of “continuationToken”
+            in the query string and return all results at once (mimicking existing behavior).
+            For servers not supporting pagination, this is the current state and they would return all values at once.
+          required: false
+          allowEmptyValue: true
+          schema:
+            type: string
+        - name: pageSize
+          in: query
+          description:
+            An upper bound on the number of results to return on the next page.
+          required: false
+          allowEmptyValue: true
+          schema:
```

Comment:
Should the format be of type uuid?

Reply:
I think the `pageToken` does have to be unqiue, however when checking a pagination token example from aws, they just mention a unique string without mentioning the format being `uuid`.
https://docs.aws.amazon.com/step-functions/latest/apireference/API_ListActivities.html

```
[nextToken](https://docs.aws.amazon.com/step-functions/latest/apireference/API_ListActivities.html#API_ListActivities_RequestSyntax)
If nextToken is returned, there are more results available. The value of nextToken is a unique pagination token for each page. Make the call again using the returned token to retrieve the next page. Keep all other arguments unchanged. Each pagination token expires after 24 hours. Using an expired pagination token will return an HTTP 400 InvalidToken error.

Type: String

Length Constraints: Minimum length of 1. Maximum length of 1024.

Required: No

```